arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -554,7 +639,30 @@
     },
     "MediaService": {
       "description": "The properties of a Media Service resource.",
+      "example": {
+        "location": "West US",
+        "properties": {
+          "storageAccounts": [
+            {
+              "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+            "isPrimary": true
+            }
+          ]
+        }
+      },
       "properties": {
+        "name": {
+          "description": "Then name of the Media Service resource.",

```

Comment:
@BrianBlum - MediaService is inheriting (allOf) from TrackedResource which in turn is inheriting (allOf) from Resource.
Resource has name, id, type defined as its properties. Then you cannot redefine them in the child class. Please remove them.